src/components/modals/CreateAntModal/CreateAntModal.tsx

```diff
@@ -365,7 +365,7 @@ function CreateAntModal({ show }: { show: boolean }) {
                               dataIndex: 'attribute',
                               key: 'attribute',
                               align: 'left',
-                              width: isMobile ? '0px' : '30%',
+                              width: isMobile ? '0px' : '50px',
```

Comment:
moving from % to hard coded pixels concerns me. what's the intent?

Reply:
txid inputs were getting squished too much, can change back to a smaller percentage.

Reply:
changed back to a percentage, but lowered it